src/test/java/com/opentok/test/OpenTokTest.java

```diff
@@ -832,16 +1033,12 @@ public void testStartComposedArchive() throws OpenTokException {
         ArchiveProperties properties = new ArchiveProperties.Builder()
                 .outputMode(OutputMode.COMPOSED)
                 .build();
-
         Archive archive = sdk.startArchive(sessionId, properties);
         assertNotNull(archive);
         assertEquals(sessionId, archive.getSessionId());
         assertNotNull(archive.getId());
         assertEquals(OutputMode.COMPOSED, archive.getOutputMode());
-
         verify(postRequestedFor(urlMatching(archivePath)));
-                // TODO: find a way to match JSON without caring about spacing
```

Comment:
I don't think we should be removing these comments

Reply:
Adding 
                // TODO: find a way to match JSON without caring about spacing 
                //.withRequestBody(matching(".*"+".*")) 
in one place at the top of start archive method